src/test/java/org/prebid/server/it/MinuteMediaTest.java

```diff
@@ -0,0 +1,37 @@
+package org.prebid.server.it;
+
+import io.restassured.response.Response;
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.prebid.server.model.Endpoint;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static java.util.Collections.singletonList;
+
+@RunWith(SpringRunner.class)
+public class MinuteMediaTest extends IntegrationTest {
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromMinuteMedia() throws IOException, JSONException {
+        // given
+
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/minutemedia-exchange"))
```

Comment:
As you added query param to test application props pls add `withQueryParam` check